.eslintignore

```diff
@@ -0,0 +1,7 @@
+coverage
+/traversal
```

Comment:
hm, i'd expect gitignored things to be already eslintignored. they're not?

Reply:
Let me add `--ignore-path .gitignore` to the npm run script instead of checking in this file. VSCode will skips linting anything that's gitignored anyway, and we won't have to maintain ignored files in multiple places.